book/website/requirements.txt

```diff
@@ -5,4 +5,4 @@ numpy
 datascience
 folium
 matplotlib
-jupyter-book==0.5.2
+jupyter-book
```

Comment:
0.6.4 is the version I used to update the book, so we should pin it to that until we feel confident in making future upgrades

Reply:
great, I can pin it to that version